resources/js/app.tsx

```diff
@@ -12,6 +12,7 @@ import { Ziggy } from './ziggy';
 
 // @ts-expect-error -- we're injecting this on purpose
 globalThis.Ziggy = Ziggy;
+globalThis.mediaUrl = import.meta.env.VITE_MEDIA_URL;
```

Comment:
should this just be `MEDIA_URL`?
https://github.com/RetroAchievements/RAWeb/blob/c00a8dc4a55c613c6722ea0b34fca8d1d7c2a284/config/app.php#L68
https://github.com/RetroAchievements/RAWeb/blob/c00a8dc4a55c613c6722ea0b34fca8d1d7c2a284/.env.example#L33

Mine is currently set to:
```
MEDIA_URL=https://media.retroachievements.org
```